draft-ietf-quic-transport.md

```diff
@@ -7328,12 +7328,12 @@ permanent registrations.
 
 The creation of a registry MAY identify a range of codepoints where
 registrations are governed by a different registration policy.  For instance,
-the registries for 62-bit codepoints in this document have stricter policies for
-codepoints in the range from 0 to 63.
+the frame type registry in {{iana-frames}} has a stricter policy for codepoints
```

Comment:
The transport error also have this range, but then have an additional range so I think this text need tweaking.

Reply:
Not sure that I follow.  Frame types have a stricter policy for 0..=63.  I don't see why would need to talk about the other ~2^62 values.

Reply:
I agree. @gloinul ?